spec/features/user/user_can_place_their_ship_spec.rb

```diff
@@ -7,14 +7,30 @@
       player_1_board = Board.new(4)
       player_2_board = Board.new(4)
       game = Game.create(id: 64, player_1_id: user.id,
-      player_2_id: user_2.id, player_1_board: player_1_board, player_2_board: player_2_board)
+        player_2_id: user_2.id, player_1_board: player_1_board, player_2_board: player_2_board)
+
       conn = Faraday.new(:url => 'http://localhost:3000')
 
       response = conn.post do |req|
-        req.url "/api/v1/games/64/ships"
+        req.url "/api/v1/games"
+        req.headers['Content-Type'] = 'application/json'
+        req.headers['X-API-Key'] = ENV['BATTLESHIFT_API_KEY']
+        req.body = { 'opponent_email': ENV['BATTLESHIFT_EMAIL'] }.to_json 
+      end
+
+      expect(response).to be_success
+      game_json = JSON.parse(response.body, symbolize_names: true)
+
+      response = conn.post do |req|
+        req.url "/api/v1/games/#{game_json[:id]}/ships"
         req.headers['Content-Type'] = 'application/json'
-        req.headers['X-API-Key'] = user.api_key
-        req.body = { 'email': ENV['BATTLESHIFT_OPPONENT_EMAIL'] }.to_json
+        req.headers['X-API-Key'] = ENV['BATTLESHIFT_API_KEY']
```

Comment:
We don't need the ship payload information coming through in just placing the ships